zbar/image.h

```diff
@@ -162,9 +162,9 @@ static inline zbar_image_t *_zbar_image_copy(const zbar_image_t *src,
 	/* Do it word per word, in order to speedup */
 	for (i = 0; i < len; i += sizeof(long))
```

Comment:
Could you fix this loop and make it run `len / sizeof(long)` times?

Reply:
Yes, I updated locally, pushing to this branch right now.